otter/models/interface.py

```diff
@@ -40,8 +42,8 @@ class GroupState(object):
     TODO: ``remove_active``, ``pause`` and ``resume`` ?
     """
     def __init__(self, tenant_id, group_id, group_name, active, pending,
-                 group_touched, policy_touched, paused, status, desired=0,
-                 now=timestamp.now):
+                 group_touched, policy_touched, paused, status,
+                 error_reasons=[], desired=0, now=timestamp.now):
```

Comment:
Non-blocking - since we're just creating new lists and assigning the error reasons anyway (we're not mutating anything), this is fine, but just for the sake of making sure nothing gets broken by accident in the future, can we set `error_reasons` to a default of `()` instead?